src/utilities/confusedMessageGenerator.js

```diff
@@ -0,0 +1,17 @@
+function confusedMessageGenerator(possibleLanguages, reactionEmojies) {
```

Comment:
rename this file to kabab case to match with the other files and export it on the same line, again to match the other files so far.

Add comments/docs on what each of the params are, represent and their data types.

Reply:
Done renaming the file and adding JSdoc. Do you also want me to rename the function name to `kabab case`?